org/ystia/ansible/linux/ansible/playbooks/create.yml

```diff
@@ -62,5 +60,6 @@
           - "ansible=={{ANSIBLE_VERSION}}"
           - "jmespath==0.9.4"
           - "netaddr==0.7.19"
+          - "paramiko"
```

Comment:
does it worthwhile using a specific version of paramiko here in order to prevent breaking changes?

Or something >=min version & <new major version?

Reply:
I was thinking to do it first, but didn't find a version referenced in Ansible doc. For example the latest install guide at https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html#installing-ansible-with-pip just says:
```
In order to use the paramiko connection plugin or modules that require paramiko, install the required module:
$ python -m pip install --user paramiko
```
So I left it this way

Reply:
ok